samtranslator/model/eventsources/push.py

```diff
@@ -484,6 +484,23 @@ def to_cloudformation(self, **kwargs):
         return resources
 
 
+class AlexaSmartHome(PushEventSource):
+    resource_type = 'AlexaSmartHome'
+    principal = 'alexa-connectedhome.amazon.com'
+
+    property_types = {}
```

Comment:
Shouldn't this have an 'EventSourceToken' Property?

Reply:
Can you update the existing Alexa Skill with this also?